plugins/connectrpc/dart/v0.4.0/buf.plugin.yaml

```
@@ -0,0 +1,12 @@
+version: v1
+name: buf.build/connectrpc/dart
+plugin_version: v0.4.0
+source_url: https://github.com/connectrpc/connect-dart
+integration_guide_url: https://connectrpc.com/docs/dart/getting-started
+description: Generates client stubs for connectrpc. Compatible with the gRPC, gRPC-Web, and Connect RPC protocols.
+deps:
+  - plugin: buf.build/protocolbuffers/dart:v21.1.2
```

```suggestion
  - plugin: buf.build/protocolbuffers/dart:v22.0.1
```

@srikrsna-buf - Is it ok to bump this dependency to the latest?

---

As this is used for gen SDKs this should be fine.